packages/graphback-cli/src/helpers/common.ts

```diff
@@ -1,12 +1,27 @@
-import { accessSync } from 'fs'
+import { accessSync, readFileSync } from 'fs'
+import { join } from 'path'
 import { logError } from '../utils'
 
 
-export const checkDirectory = (): void => {
-  try{
-    accessSync(`${process.cwd()}/model`)
-  } catch(err) {
-    logError(`model directory not found. Make you sure you are in the root of your project.`)
+export const checkDirectory = (): string => {
+
+  let pathForModel
+
+  try {
+
+    const configPath = `${process.cwd()}/graphback.json`
```

Comment:
`${process.cwd()}/graphback.json` is used in couple places maybe we can setup it as constant in separate folder and simply require it?